### DISCUSSION TITLE
How to get menu on other folders

### DISCUSSION THREAD

**Discussion by mkovnick** [onweru/compose#72] [Q&A]
Hi, I am new to hugo, so forgive the question is it may be simple.  I noticed that I get the side menu for any file I put in the "Docs" folder, but not in other folders.   I am trying to also make a "Tutorials" folder that would **also** have a side menu and I cannot figure out how to do it.  Hoping for help :-)

**Comment by onweru** [comment]
Thank you @mkovnick, for raising this issue. I have updated the theme in #73  and [ documented the change here](https://docs.neuralvibes.com/docs/compose/organize-content/#documentation).

**Reply by mkovnick** [reply]
Thanks!   I will try now :-)

**Reply by onweru** [reply]
Cool, let me know how it goes.
